examples/checkable-item.php

```diff
@@ -22,8 +23,10 @@
     })
     ->addSubMenu('Interpreted', function (CliMenuBuilder $b) use ($itemCallable) {
         $b->setTitle('Interpreted Languages')
-            ->setUncheckedMarker('[○] ')
-            ->setCheckedMarker('[●] ')
+            ->setCheckableStyle(function (CheckableStyle $style) {
```

Comment:
This is irking me a little. Can we change `CheckableItem` to CheckboxItem` to match `RadioItem` please, then possibly change this to `CheckboxStyle` and `->setCheckboxStyle`. Can we rename the checkbox as a separate PR though. This one in a bit unruly!

Reply:
Yes, removing the `able` from items makes sense and it was bothering me as well.

As for this being unruly - I can yank out the bits with `Closure` that fixes nested styles not being applied, and put it into a separate PR. Anything else you think isn't clear, though?

Reply:
That would be very helpful, I'll try pick out and comment on things I think can be separated. Appreciate all the work though!